webcomponents/tool/src/main/frontend/packages/sakai-image-editor/src/SakaiImageEditor.js

```diff
@@ -44,12 +44,11 @@ export class SakaiImageEditor extends SakaiDialogContent {
 
   done() {
 
-    this.cropper.getCroppedCanvas().toBlob(blob => {
-
+    const croppedCanvas = this.cropper.getCroppedCanvas({ maxWidth: 1920, maxHeight: 1080 });
+    croppedCanvas.toBlob(blob => {
       const url = URL.createObjectURL(blob);
       this.dispatchEvent(new CustomEvent("image-edited", { detail: { url, blob }, composed: true, bubbles: true }));
-    });
-
+    }, "image/webp", 0.75);
```

Comment:
this should produce a nicely optimized image compared to the previous lossless PNG at full resolution

Reply:
.9 seem to sufficient and reduces the image significantly